app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,105 @@
-// Code your JavaScript / jQuery solution here
+const WINNING_COMBOS = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]];
+var currentGame = 0,
```

Comment:
Prefer to use `let` or `const` over var. For more info, check out this link: https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e